media/css/externalpages/pocket/style.scss

```diff
@@ -7,9 +7,11 @@
 
 @import 'utils/variables';
 @import 'utils/fonts';
+@import 'utils/tokens';
 
 @import 'components/nav';
 @import 'components/footer';
+@import 'components/add';
```

Comment:
I think this could be a longer conversation about how we wanted to handle our css bundles, but i was thinking that `styles.scss` would have our global styles, fonts, variables, and tokens and components would be added to a bundle independently. Happy to talk more about this though!